src/components/my-journey/Contests.tsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
"Line 27: Make the equality sign triple ===; otherwise, it shows a warning."

Reply:
`Line 19` I feel that it would be better if the skeleton code were extracted as a component. It is kind of confusing to see div tags without any children or text inside them.

Reply:
`Line 45` typo-error _occurred_ and make the word contests in lowercase

Reply:
`Line 28:` Shouldn't the text '`No Upcoming Contests`' be in sentence case?